sdk/identity/azure-identity/azure/identity/_credentials/authorization_code.py

```diff
@@ -30,6 +30,9 @@ class AuthorizationCodeCredential(GetTokenMixin):
         the authority for Azure Public Cloud (which is the default). :class:`~azure.identity.AzureAuthorityHosts`
         defines authorities for other clouds.
     :keyword str client_secret: One of the application's client secrets. Required only for web apps and web APIs.
+    :keyword list[str] additionally_allowed_tenant_ids: optional additional tenant ids for which the credential
+            may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for
+            any tenant the application is installed.
```

Comment:
nit: the current language is a little confusing (at least, to me)
```suggestion
            may acquire tokens. Add the wildcard value "*" to allow the credential to attempt to acquire tokens
            for any tenant.
```